modules/cluster-logging-collector-legacy.adoc

```diff
@@ -0,0 +1,240 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-legacy_{context}"]
+= Forwarding logs using the legacy configuration method
+
+You can configure cluster logging to send logs to destinations outside of your {product-title} cluster instead of the default Elasticsearch log store using Fluentd or syslog by creating a configuration file and ConfigMap.
```

Comment:
This section is referring to unmanaged mode right? It should say something about that.

Reply:
@alanconway As far as I know, unmanaged is not required. The process creates a new configMap and doesn't affect any existing objects. 
@jcantrill Can you confirm?

Reply:
@mburke5678  Yes, unmanaged is not required